my_robot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 20  
  
    # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint1 
    pid: {p: 100.0, i: 0.01, d: 10.0}

  joint4_position_controller:
    type: effort_controllers/JointPositionController #can chagne to JointVelocityController
    joint: joint4
    pid: {p: 100.0, i: 1, d: 10.0}

  joint15_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint15
    pid: {p: 100.0, i: 1, d: 10.0}